src/System.Management.Automation/AssemblyInfo.cs

```diff
@@ -2,9 +2,6 @@
 using System.Reflection;
 using System.Resources;
 
-[assembly: AssemblyFileVersionAttribute("3.0.0.0")]
-[assembly: AssemblyVersion("3.0.0.0")]
```

Comment:
I think bumping assembly version number needs to be reviewed by powershell committee before the change.